addons/website_forum/models/forum_post.py

```diff
@@ -260,7 +260,7 @@ def _compute_post_karma_rights(self):
             post.can_comment = is_admin or user.karma >= post.karma_comment
             post.can_comment_convert = is_admin or user.karma >= post.karma_comment_convert
             post.can_view = is_admin or user.karma >= post.karma_close or (post_sudo.create_uid.karma > 0 and (post_sudo.active or post_sudo.create_uid == user))
-            post.can_display_biography = is_admin or post_sudo.create_uid.karma >= post.forum_id.karma_user_bio
+            post.can_display_biography = is_admin or (post_sudo.create_uid.website_published and post_sudo.create_uid.karma >= post.forum_id.karma_user_bio)
```

Comment:
I would have added it at the end to ease reading the diff but that is really marginal and non-blocking.

```suggestion
            post.can_display_biography = is_admin or (post_sudo.create_uid.karma >= post.forum_id.karma_user_bio and post_sudo.create_uid.website_published)
```

Reply:
@flch-odoo updated the PR

Reply:
hi @flch-odoo ,
is it okay to merge now...
thanks